ftl/ftl_node_integration_tests_yaml.py

```diff
@@ -30,11 +32,12 @@ def main():
     }
 
     # Generate a set of steps for each test and add them.
-    for test in ['gcp_build_test', 'packages_test', 'packages_lock_test']:
-        cloudbuild_yaml['steps'] += run_test_steps(
-            _NODE_BASE,
-            '%s-image' % test,
-            test)
+    for test in [
+            'gcp_build_test', 'packages_test', 'packages_lock_test',
+            'destination_test'
```

Comment:
This loop won't quite work, since you need special args for destination-app passed to ftl:node_builder_image.